sys/include/suit/transport/worker.h

```diff
@@ -52,6 +94,20 @@ void suit_worker_trigger(const char *url, size_t len);
  */
 int suit_handle_url(const char *url);
 
+/**
+ * @brief   Trigger a SUIT update on an in-memory manifest
+ *
+ * @note Make sure the thread calling this function has enough stack space to fit
+ *       a whole flash page.
+ *
+ * @param[in] buffer     Memory area containing a SUIT manifest.
+ * @param[in] size       Size of the manifest in @p buffer.
+ *
+ * @return 0 on success
+ *        <0 on error
+ */
+int suit_handle_prepared(const uint8_t *buffer, size_t size);
```

Comment:
Ok I'm beginning to like this as a more use-friendly alternative to calling `suit_parse()` - it can be used independently from the 'prepared'/worker thread machinsm.

Maybe call it `suit_handle_manifest_buf()` instead?

Reply:
Good point, done.